components/iam/pkg/server/server.go

```diff
@@ -35,7 +35,7 @@ func Start(logger *logrus.Entry, version string, cfg *config.ServiceConfig) erro
 	}
 
 	// Requests to /oidc/* are handled by oidc.Router
-	oidcService := oidc.NewOIDCService()
+	oidcService := oidc.NewService()
```

Comment:
This is an example where the package already provides the `OIDC` context, and hence not needed in the struct name.